resources/views/auth/register.blade.php

```diff
@@ -6,6 +6,17 @@
         ></script>
     @endsection
 
+    <x-input-error
+        :messages="$errors->github->get('github_username')"
+        class="mb-5"
+    />
+
+    @if ($githubUsername)
+        <div class="mb-5">
```

Comment:
If we have received the email from GitHub, we invite the user to fill in the rest of their details